src/services/docker-service.ts

```diff
@@ -526,27 +566,32 @@ export class DockerService {
           }
 
           DockerService.debug(output)
+
+          const errorMsg= debugEnabled ? errUnableToExtractDigestMsg : `${errUnableToExtractDigestMsg}${enableDebugRetrievedContentMsg}`
```

Comment:
should we add a white space between `${errUnableToExtractDigestMsg}${enableDebugRetrievedContentMsg}`?

Reply:
IMHO The space between the two strings variables it's not necessary in this case because the first string have a "\n" at the end.